vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -65,11 +68,12 @@ class MSBuildTools {
 
     // Set platform toolset for VS 2019
     if (this.version === '16.0') {
+      // we are guaranteed to have Microsoft.VisualStudio.ComponentGroup.UWP.VC.v141
+      const vsPath = path.resolve(this.path, '../../..');
+      const targetsPath = path.join(vsPath, 'MSBuild\\Microsoft\\VC\\v150\\');
```

Comment:
This value is still hard-coded.
If MSVC `v142` is used, it will fail.
VCTargetsPath should be a function of the platform toolset (i.e. `v141`), ideally.